### ISSUE TITLE
abilities data

### ISSUE THREAD

**Issue by acstanton515** [aoe4world/data#15]
put together an abilities data source. would it be possible to get it from the game files?

- en - network of castles/citadels
- all - camel unease (effects all civs cav)
- de - tower of victory
- ot - various mehter bonuses
- fr - royal knight bonus damage after charge
- mo - khan abilities

etc.

**Comment by robertvanhoesel** [comment]
Yeah this is on my backlog and yes all of this is somewhere in the game files, the challenging thing about it is that there are various types:
- Passive and active influences that affect unit/building
- Passive and active influences that unit can trigger
- Normal unit abilities (set up camp, scout eagle, paviche etc)

Some of the passive influences on buildings are labelled on the buildings (i.e. 'In range of town center can heal' 'adds to golden age'), and some on the building that triggers it.

A plain list of all influences and abilities is possible, but correct correlation between units and effects is less easy. Camel Unease is a good example of that, should it show on all other cav civs that they can have this effect? What about team game things, like Kurultai and Abbasid Keep Heeling, etc, should that show on all units that can get healed by it? You see where I'm going :D 

Suggestions welcome!

Did you have it in mind for something specific?

**Comment by acstanton515** [comment]
> Yeah this is on my backlog and yes all of this is somewhere in the game files, the challenging thing about it is that there are various types:
> 
> * Passive and active influences that affect unit/building
> * Passive and active influences that unit can trigger
> * Normal unit abilities (set up camp, scout eagle, paviche etc)
> 
> Some of the passive influences on buildings are labelled on the buildings (i.e. 'In range of town center can heal' 'adds to golden age'), and some on the building that triggers it.
> 
> A plain list of all influences and abilities is possible, but correct correlation between units and effects is less easy. Camel Unease is a good example of that, should it show on all other cav civs that they can have this effect? What about team game things, like Kurultai and Abbasid Keep Heeling, etc, should that show on all units that can get healed by it? You see where I'm going :D
> 
> Suggestions welcome!
> 
> Did you have it in mind for something specific?

Absolutely see where you are going with those questions; as I revisit this, I'll build some JSON data model.

I've got something in mind, but not much to show yet.
